### PR TITLE
feat(validation) Support for Validation Array fixes #41

### LINKED ISSUE #41: Support for Validation to be An Array of strings

### PR DESCRIPTION
**Pull request by mohammedzamakhan**
This will make sure that Validations can be in an array, and thus JSON can be used to populate the form with validations.

### CODE REVIEW

File: src/templates/field.ts [hunk 1]
```diff
@@ -24,7 +27,14 @@ export abstract class Field implements OnInit {
 
   get formControl(): AbstractControl {
     if (!this._control) {
-     this.createControl();
+      if (typeof this.field.validation === "object") {
```
Comment (aitboudad) [inline src/templates/field.ts]: I would prefer to move this into `FormlyField` component

Comment (mohammedzamakhan) [inline src/templates/field.ts]: Makes sense. Thanks 😄

Comment (mohammedzamakhan) [inline src/templates/field.ts]: Done 💯

File: src/components/formly.field.ts [hunk 2]
```diff
@@ -55,6 +56,13 @@ export class FormlyField extends FormlyCommon implements OnInit {
 
   createChildFields() {
     if (this.field && !this.field.template && !this.field.fieldGroup) {
+      if (typeof this.field.validation === "object") {
```
Comment (aitboudad) [inline src/components/formly.field.ts]: can we use `Array.isArray(this.field.validation)` instead ?

Comment (mohammedzamakhan) [inline src/components/formly.field.ts]: Done 💯

**Pull request merged**